pkg/tsdb/azuremonitor/applicationinsights-metrics_test.go

```diff
@@ -175,7 +175,7 @@ func loadInsightsMetricsResponse(name string) (MetricsResult, error) {
 	var mr MetricsResult
 
 	path := filepath.Join("testdata", name)
-	f, err := os.Open(path)
+	f, err := os.Open(filepath.Clean(path))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.